editoast/src/views/timetable.rs

```diff
@@ -30,25 +29,30 @@ use crate::models::prelude::*;
 use crate::models::timetable::Timetable;
 use crate::models::timetable::TimetableWithTrains;
 use crate::models::train_schedule::TrainSchedule;
-use crate::models::train_schedule::TrainScheduleChangeset;
 use crate::models::Infra;
 use crate::views::train_schedule::train_simulation_batch;
-use crate::views::train_schedule::TrainScheduleForm;
 use crate::views::train_schedule::TrainScheduleResult;
 use crate::views::AuthenticationExt;
 use crate::views::AuthorizationError;
 use crate::AppState;
 use crate::RetrieveBatch;
 use editoast_models::DbConnectionPoolV2;
+use editoast_schemas::train_schedule::TrainScheduleBase;
+use crate::models::train_schedule::TrainScheduleChangeset;
+use crate::views::train_schedule::TrainScheduleForm;
+
+use super::pagination::PaginatedList as _;
+use super::pagination::PaginationQueryParams;
+use super::pagination::PaginationStats;
 
 crate::routes! {
     "/timetable" => {
         post,
         "/{id}" => {
             delete,
-            get,
-            "/conflicts" => conflicts,
+            "/train_schedules" => get,
             "/train_schedule" => train_schedule,
```

Comment:
Not sure how easy it would be, but it'd be great to have both GET and POST routes under `/train_schedules`. That'll probably imply some changes to the frontend since the generated function name will change a bit, but since there's already front changes in this PR, maybe this could be done here?